---
layout: page
title: My Research
tags: [about, research]
modified: 
comments: false
share: false
image:
  feature: header.png
  credit: 
  creditlink: 
---

Welcome to my academic website. Here you can find information about my research, publications, and other news (such as museum expiditions). I will also be focused on I'm currently a doctoral student in the [Miller Lab](http://mj-miller.net/) in the Department of Ornithology at the Sam Noble Oklahoma Museum of Natural History and the Department of Biology at the the University of Oklahoma (OU). Prior to OU, I earned a Masters degree in [Jeff Peters Lab](http://people.wright.edu/jeffrey.peters) at Wright State University in Dayton, Ohio. My thesis was doing population genomics in the Australian Teals  (in the genus _Anas_). Prior to my Masters Degree, I earned a Bachelor of Arts in Zoology and Latin American Studies. I also worked did population genetics in _Carica papaya_ in the [Moore Lab](https://miamioh.edu/cas/academics/departments/biology/about/faculty/moore/index.html).    
&nbsp; 

### Primary research interests:  

* Genomics: ddRADseq bioinformatics and phylogenetics  
* Ecology: Interspecific reproductive interference in hyperdiverse lineages  
* Evolution: Comparative approaches to measuring genomic introgression 
* Botany: Pollen-pistil relationships and floral evolution in _Pedicularis_  
&nbsp; 

<a href="/images/deren_fieldwork.jpg"><img src="/images/deren_fieldwork.jpg"></a>

&nbsp; 

## Topics  

__Bioinformatics__ -- I created and still actively develop the program _pyrad_, which is used to assemble genomic RADseq data sets. I love learning about Python programming and linux, and I'm a huge advocate for open-source code and data sharing. You can follow active development of my projects on [github](github.com/dereneaton).  

&nbsp; 

__Reproductive Interference__ -- My research is broadly focused on understanding why some clades of organisms contain more species than others, and within this context Iâ€™m especially interested in the long-recognized relationship among flowering plants where families with more specialized pollination mechanisms tend to contain more species. To understand this pattern I study how flower evolution affects rates of speciation, and reciprocally, how interactions among co-occurring species, especially within highly diverse clades, affect flower evolution.


<figure class="half">
	<a href="/images/integ-1024x680.jpg"><img src="/images/integ-1024x680.jpg"></a>
        <a href="/images/siphonantha_bee.jpg"><img src="/images/siphonantha_bee.jpg"></a>	
</figure>    


&nbsp; 

&nbsp; 


__Pedicularis__ -- Much of my research is focused on the hyperdiverse genus _Pedicularis_ in its center of diversity in the Hengduan Mountains of China (the eastern side of the Tibetan plateau), where nearly 300 species are endemic. _Pedicularis_ in this region frequently overlap in their geographic ranges, flower synchronously during the short alpine summer, and share bumblebee pollinators. This combination of factors raises the potential for pollen transfer _between_ different species, which can have negative fitness consequences (termed reproductive interference), and consequently impose strong selection on floral phenotypes. I combine genomic RADseq approaches above and below the species-level with field-based experiments and ecological studies to better understand the evolutionary consequences of these interactions.  

+ collaborators: [Richard Ree](http://www.reelab.net), 
  		 	   [Shuang-Quan Huang](http://www.researchgate.net/profile/Shuang-Quan_Huang/),
			   		 [Yu Wen-bin](http://wbyu.wikispaces.com/home)   

<a href="/images/jiambda_landscape.jpg"><img src="/images/jiambda_landscape.jpg"></a>  

&nbsp; 

&nbsp; 


__Oaks__ -- Coyne and Orr described oaks as "a worst case scenario of the biological species" due to their often blurred species boundaries resulting from hybridization. Whether as a consequence of hybridization, insufficiently variable markers, or sorting of ancestral polymorphisms in large populations of long lived trees, previous phylogenetic studies have shown great difficulty in resolving oak relationships at both shallow and deep evolutionary timescales. Applying genomic RADseq data we have resolved a backbone phylogeny of the American clade oaks (Hipp et al. 2014) and shown evidence of frequent hybridization among the live oaks which affects phylogenetic inference (Eaton et al. 2015), and which has likely persisted over millions of years (Cavender-Bares et al. 2015).  

+ collaborators: [Jeannine Cavender-Bares](http://www.cbs.umn.edu/research/research-cbs/faculty-labs/cavender),
  		 	    [Andrew Hipp](http://systematics.mortonarb.org/lab/)  

<a href="/images/angel_oak.jpg"><img src="/images/angel_oak.jpg"></a>  


&nbsp; 

&nbsp; 



__Viburnum__ -- A model system for phylogenetic comparative methods in plants, and in particular historical biogeography and leaf trait evolution. We are using RADseq methods to infer deep-scale phylogenies, infer introgression between lineages, resolve polyploid lineages, and study replicate radiations of leaf forms among Viburnum in central and south America. 

+ collaborators: [Michael Donoghue](http://www.donoghuelab.yale.edu). 

<a href="Leaf shape and size variation in a Viburnum hybrid zone in Mexico"><img src="/images/Vib_Mexico_leaves.jpg"></a>  

&nbsp; 

&nbsp; 


